bin/standardize

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 bundle exec standardrb --fix
-cd ./javascript && yarn run prettier --write cable_ready.js
+cd ./javascript && yarn run prettier --write cable_ready.js
```

Comment:
I don't understand how this is different. I don't think the file had a newline before?